install_ddev.sh

```diff
@@ -14,26 +14,39 @@ BINOWNER=$(ls -ld /usr/local/bin | awk '{print $3}')
 USER=$(whoami)
 SHACMD=""
 FILEBASE=""
+CURRENT_DIR=$PWD
 
 if [[ "$OS" == "Darwin" ]]; then
     FILEBASE="ddev_macos"
+
+    if ! docker --version >/dev/null 2>&1; then
```

Comment:
Let's give people a chance here before just installing on their machine. Can we give them a Y/N option before proceeding? docker is great, but they need to agree to installing it. The choice could be here, or perhaps we could do it earlier.

Are you going to push this on Windows as well?